quinn/src/connection.rs

```diff
@@ -44,7 +44,7 @@ impl Connecting {
     /// intercepted by a man-in-the-middle. If this occurs, the handshake will not complete
     /// successfully.
     ///
-    /// Once [`Connection::is_handshaking`] returns false, further transmissions are unaffected by
+    /// Once [`Session::is_handshaking`] returns false, further transmissions are unaffected by
```

Comment:
This isn't part of the public API of the this higher-level crate, so I don't think we should mention it here.

Reply:
By that do you mean remove the mention altogether? Or do you mean that eventually a `Connection::is_handshaking` method will be added?

Reply:
I assume its the second so I'm gonna leave it be.

Reply:
I mean this paragraph should be deleted. Its place should ultimately be taken by documentation for the hypothetical future that completes when the connection is established and indicates 0-RTT acceptance or rejection, but that doesn't need to be handled in this PR.

Reply:
I suppose there's no strong reason we couldn't have an accessor too, if it's useful to someone.

Reply:
Can't say I would use such a method. I guess I'll remove this part of the doc, while I'm at it.